llvm/tools/llvm-readtapi/llvm-readtapi.cpp

```diff
@@ -151,8 +171,10 @@ int main(int Argc, char **Argv) {
         exit(1);
       });
   if (Args.hasArg(OPT_help)) {
-    Tbl.printHelp(outs(), "llvm-readtapi [options] <inputs>",
-                  "LLVM TAPI file reader and manipulator");
+    Tbl.printHelp(
+        outs(),
+        "llvm-readtapi [options]* [-arch <arch>]* <inputs> [-o <output>]*",
```

Comment:
Isn't the `*` implied when `[options]` is used? There can be multiple outputs?

Reply:
I guess I used * to denote 0 or more flags, in the case for output, no -o means write to stdout. But yes you're right, I don't need one for [options].